design/ceph/object/store.md

```diff
@@ -70,6 +72,11 @@ If `preservePoolsOnDelete` is set to 'true' the pools used to support the object
     erasureCoded:
       dataChunks: 6
       codingChunks: 2
+  radosNamespaces:
+    # If true, create RADOS namespaces for this object store.
+    # Default is false, preserve the current behavior for backward compatibility.
+    create: true
```

Comment:
I don't think `create` is a great value for this. Can we not use the presence of `poolName` below to indicate that the option is enabled?